### PR TITLE
use ver-update to update pkg/eve/Dockerfile.in

### PR DESCRIPTION
**Pull request by dkionka**
add build-tools/ver-update/README.md with instructions

### CODE REVIEW

**Comment by dkionka** [comment]
No point having new scripts without a step-by-step example.

File: pkg/eve/Dockerfile.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.12 as tools
+FROM alpine:3.13.0 as tools
```
Comment (rvs) [inline pkg/eve/Dockerfile.in:1]: Can you please drop this change from the PR @dkionka ?

**Review by rvs — CHANGES_REQUESTED** [review]
One unrelated dockerfile change -- looks good otherwise

File: pkg/eve/Dockerfile.in (hunk 1 above)
Comment (dkionka) [reply pkg/eve/Dockerfile.in]: Done.

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**